src/tests/ftest/vmd/fault_reintegration.yaml

```diff
@@ -0,0 +1,79 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+# If we define the server under test_servers, launch.py will convert it to the
+# actual server name passed into --test_servers. If above test_servers is hosts,
+# it'll be used as one of the servers at test startup time, so use something
+# other than hosts.
+timeout: 1000
+setup:
+  start_servers_once: false
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      env_vars:
+        - DD_MASK=all
+        - D_LOG_MASK=DEBUG
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+        1:
+          class: nvme
+          bdev_list: ["aaaa:aa:aa.a"]
+          bdev_class: nvme
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem1"]
+          scm_mount: /mnt/daos1
+        1:
+          class: nvme
+          bdev_list: ["bbbb:bb:bb.b"]
+          bdev_class: nvm
```

Comment:
```suggestion
      storage: auto
```

Reply:
Thanks for the comment. 
I have also removed the env_vars from the test.yaml which might cause build error on Leap15.